scripts/gitlab/build_and_test.sh

```diff
@@ -111,6 +112,17 @@ then
     # Map CPU core allocations
     declare -A core_counts=(["lassen"]=40 ["ruby"]=28 ["corona"]=32)
 
+    # If using Multi-project, set up the submodule
+    if [[ -n ${raja_version} ]]
+    then
+      cd tpl/RAJA  
+      echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+      echo "~~~~ Updating RAJA Submodule to develop ~~~"
+      echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+      git pull origin develop
```

Comment:
will this also pull the RAJA submodule versions that are used in RAJA develop? just want to make sure this is what's happening.

Reply:
Good point - I think we need to add a submodule update here too.